hs_file_types/models/netcdf.py

```diff
@@ -414,6 +399,12 @@ def set_file_type(cls, resource, user, file_id=None, folder_path=''):
                                      res_type_specific_meta, file_type_metadata, resource)
 
                 # create the ncdump text file
+                dump_file_name = nc_file_name + "_header_info.txt"
+                for file in resource.files.filter(file_folder=folder_path):
+                    fname = os.path.basename(file.resource_file.name)
+                    if fname in dump_file_name:
+                        file.delete()
+                        break
```

Comment:
```if fname in dump_file_name: file.delete()``` looks a bit problematic to me. What is the reason this has to be done? It'd be good to add a comment here for some clarification.

Reply:
This was a bug that took me awhile to figure out.  This logic simply deletes an existing ncdump text file if it already exists before trying to create it below.  Before this change, if you tried to set nc file type when this file existed already, the process would fail.  This is especially important for bag ingestion, but could also happen if someone were to uploaded netcdf files along with the header dump and then tried to aggregate them.

Reply:
I'll add a comment.